8.5-java11/init_container.sh

```diff
@@ -48,6 +48,11 @@ fi
 if [ -z "$WEBSITE_INSTANCE_ID" ]
 then
     export WEBSITE_INSTANCE_ID=dev
+    # set the Env variables required to test AzMon
```

Comment:
Might be good to wrap the 4 lines with a BEGIN and END comment to make it clear that the comment is applicable to all the 4 lines.